src/Mobs/Monster.cpp

```diff
@@ -203,7 +203,7 @@ bool cMonster::TickPathFinding(cChunk & a_Chunk)
 			{
 				if ((m_Path->IsFirstPoint() || ReachedNextWaypoint()))
 				{
-					m_NextWayPointPosition = Vector3d(0.5, 0, 0.5) + m_Path->GetNextPoint();
```

Comment:
Centering the mobs is no longer Monster.cpp's responsibility and it's done inside the Pathfinder instead.